rb/lib/selenium/webdriver/common/interactions/pointer_input.rb

```diff
@@ -108,20 +108,25 @@ class PointerMove < Interaction
 
         def initialize(source, duration, x, y, element: nil, origin: nil)
           super(source)
+
           @duration = duration * 1000
           @x_offset = x
           @y_offset = y
-          @origin = element || origin
+          @origin = origin || POINTER
```

Comment:
Should be `@origin = element || origin || POINTER`

Reply:
@twalpole I'll have to check the code again to be sure, but I believe later code just calls `to_s` on it the item passed in, so it will always be a string and fall foul of
```
       from invalid argument: if 'origin' is a string, it must be either 'viewport' or 'pointer'
```
Perhaps there's a more significant change required?

Reply:
Could be the element needs to be serialized differently -- I'd need to check the code again too.

Reply:
Looking through the code it appears to correctly serialize the element, what exactly are you passing as `element` when getting this error?  Are you possibly passing a Capybara element, rather than a Selenium Element (capybara_element.native) when calling driver specific code?

Reply:
I'm not calling selenium code directly anywhere so it will have been via capybara's code.  Lemme take a look and see what's what.

Reply:
Once you call `page.driver.browser...`, you are calling Selenium code